### PR TITLE
Add compat data for GlobalEventHandlers

### PR DESCRIPTION
**Pull request by luciancic**
[GlobalEventHandlers](https://developer.mozilla.org/en-US/Web/API/GlobalEventHandlers)

### CODE REVIEW

**Comment by lucian95** [comment]
Not sure about the pointer events. Added 2 transition events because I found them in the [Transition spec](https://drafts.csswg.org/css-transitions/#interface-dom) and they work in Firefox.

File: api/GlobalEventHandlers.json [hunk 1]
```diff
@@ -0,0 +1,4927 @@
...
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "10"
+            },
+            "firefox_android": {
+              "version_added"
```
Comment (wbamberg) [inline api/GlobalEventHandlers.json]: This appears to be wrong: https://bugzilla.mozilla.org/show_bug.cgi?id=1426786

This also seems the case for all the other related pointer events in this PR.

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: The page has 35 not 55 for this and related pointer events.

https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onpointerdown#Browser_compatibility

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: Should not be experimental any more IMO.

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: Should not be experimental any more IMO.

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: Should perhaps be `true`? There does seem to be a spec. (This applies to all touch events).

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: The page says Yes, but the page does seem to be wrong here, so 👍

Comment (wbamberg) [inline api/GlobalEventHandlers.json]: This suggests that the event is also available with the standard name, which isn't what the page says. Are we sure about this?

**Comment by wbamberg** [comment]
Thanks, epic PR! I had lots of comments and questions but this a big PR.

Some general stuff:

* some of these don't have pages in MDN. Some of the pages did exist under the wrong slug, and I've moved those to the right place. But some of the pages just didn't exist at all, so it's not clear what to put for them, and the `mdn_url` link won't work for them. IDK if it's better to omit `mdn_url` in such cases or not.

* some of the pages don't include compat tables, so again it's not clear what to put for them. In general I think you've made sensible calls here. I've made a few comments where I might have put something different, but probably haven't been very consistent.

* browsers should all be sorted

* In some cases you've put `null` for events that are almost certainly well-supported. Sometimes I've complained about this, but again I haven't been very consistent with this.

File: api/GlobalEventHandlers.json [hunk 2]
```diff
@@ -0,0 +1,4927 @@
...
+              {
+                "version_added": true,
+                "alternative_name": "onwebkitanimationend"
+              }
+            ],
+            "chrome": [
+              {
+                "version_added": true,
+                "notes": "Unprefixed"
+              },
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/GlobalEventHandlers.json]: I've tested with addEventListener("webkitAnimationEnd") and that worked for me. Shrug.

File: api/GlobalEventHandlers.json [hunk 3]
```diff
@@ -0,0 +1,4927 @@
...
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/GlobalEventHandlers.json]: I'm not bothering with this. Thanks for noticing it, though.

File: api/GlobalEventHandlers.json [hunk 4]
```diff
@@ -0,0 +1,4927 @@
...
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "3.5"
+            },
+            "firefox_android": {
+              "version_added": "4"
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/GlobalEventHandlers.json]: Unclear for me, too. I'm going with true.

File: api/GlobalEventHandlers.json [hunk 5]
```diff
@@ -0,0 +1,4927 @@
...
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/GlobalEventHandlers.json]: Going with true values

**Pull request merged**